azure-pipelines.yml

```diff
@@ -9,15 +9,8 @@ trigger:
     include:
     - master
   paths:
-    exclude: # don't trigger the CI if only a doc file was changed
-    - docs/*
-    - eng/common/README.md
-    - src/benchmarks/micro/Serializers/README.md
-    - src/benchmarks/micro/README.md
-    - src/benchmarks/real-world/JitBench/README.md
-    - src/benchmarks/real-world/Microsoft.ML.Benchmarks/Input/README.md
-    - src/tools/ResultsComparer/README.md
-    - README.md
+    include:
+    - eng/Version.Details.xml
```

Comment:
Won't this mean we only trigger on the schedule updates and not on other changes? What will that mean for CI runs on PRs etc?

Reply:
This is only for the CI trigger. I did not change the PR trigger. So nothing will change for the PRs, but we will only do triggered runs when we update the version of the SDK for the release branch